test/parallel/test-http-exceptions.js

```diff
@@ -30,16 +35,17 @@ const server = http.createServer(function(req, res) {
   res.end();
 });
 
+function onUncaughtException(err) {
+  console.log(`Caught an exception: ${err}`);
+  if (err.name === 'AssertionError') throw err;
+  countdown.dec();
+}
+
+process.on('uncaughtException',
+           common.mustCall(onUncaughtException, NUMBER_OF_EXCEPTIONS));
```

Comment:
Nit, the `common.mustCall` is obsolete here, because the test will not execute `countdown.dec()` in case it would not be called and would then fail because the wrapper around the countdown would fail.